packages/base-zone/src/prepare-attenuator.js

```diff
@@ -62,7 +98,7 @@ import { M } from '@endo/patterns';
  * @param {import('@agoric/base-zone').Zone} zone
  * @param {string} uKindName
  *   The `kindName` of the underlying exo class
- * @param {(string|symbol)[]} uMethodNames
+ * @param {PropertyKey[]} uMethodNames
```

Comment:
Can this be more precise?
```suggestion
 * @param {Array<keyof U>} uMethodNames
```

Reply:
Done. Though I spelled it `(keyof U)[]`